README.md

```diff
@@ -7,3 +7,5 @@ Our voices and visibility matter. Photography and storytelling can help humanize
 ## Image Components
 
 As a full-stack engineer at TONL I optimized browsable imagery load-time by building standalone GUI components collaboratively with UX and design teams using React’s JSX extension.
+
+Examples of these GUI components are image cards used to show categorical imagery shot by some of the incredible photographers TONL hires.
```

Comment:
Below this I will place screenshots or gifs of what people will see if they run the code in their browsers.